### DISCUSSION TITLE
Accessing the Admin Panel

### DISCUSSION THREAD

**Discussion by User9511** [markshust/docker-magento#1094] [Q&A]
Hey there, I've looked around and can't an  answer for this anywhere online.

I've installed magento with the one line installation and also included the sample data which is great!

However, when I go to https://magento.test/admin I'm locked out as I don't know the username  and password.

I can see the dummy data for the admin username / password / email inside  the magento.env file.

How  can I modify / update this to my own details and then use that to enter the magento dashboard ?

Thanks!

I can't find a step-by-step  guide  on this here

**Comment by YevhenZvieriev** [comment]
Hi, @User9511 

You can always use the `bin/magento admin:user:create` to create a new admin user with your details or use my [script](https://github.com/markshust/docker-magento/pull/1040).

This command will require you  the follow params :
```
Username
Email
Firstname
Lastname
Password
```
The Username will be your login, and the Password accordingly is your password.

So, you will get access to your admin Dashboard after entering your Username and Password.

To get access to the admin panel, just disable this ```bin/magento module:disable Magento_TwoFactorAuth```, and don`t forget about dependency with ```Magento_AdminAdobeImsTwoFactorAuth```.

If you have any questions, I will help you with pleasure.

**Reply by User9511** [reply]
Hey @YevhenZvieriev  thanks so much for the response!

I've followed your advice and created a new admin account which is great.

I ran this command bin/magento module:disable Magento_TwoFactorAuth but when I  try to access the admin  panel with  my new  admin account i'm still getting the error message 

<img width="566" alt="image" src="https://github.com/markshust/docker-magento/assets/22292287/17d2f1eb-3277-40e5-9de8-4e31528fa6de">

<img width="449" alt="image" src="https://github.com/markshust/docker-magento/assets/22292287/7448b800-8312-48d3-bd8d-497dbaf36678">

**Reply by YevhenZvieriev** [reply]
Hi, @User9511 

As I said ```don`t forget about dependency with Magento_AdminAdobeImsTwoFactorAuth```.

You should disable the `Magento_AdminAdobeImsTwoFactorAuth` module before the `Magento_TwoFactorAuth`.

 ```
 1. bin/magento module:disable Magento_AdminAdobeImsTwoFactorAuth
 
 2. bin/magento module:disable Magento_TwoFactorAuth
 
 3. bin/magento setup:upgrade
 ```

After that, you can get access to the Admin Panel.

**Reply by User9511** [reply]
Still not working for some reason.

I actually just  re-did a fresh install of Magento to  see  if that would  resolve the issue.

After I did a fresh  install   I did the following  in order:

1 - bin/magento admin:user:create
2. bin/magento module:disable Magento_AdminAdobeImsTwoFactorAuth
3. bin/magento module:disable Magento_TwoFactorAuth
4. bin/magento setup:upgrade

I then navigate to  https://magento.test/admin  and when  I  enter the  username  password  that I've just created I get  this error:

<img width="448" alt="image" src="https://github.com/markshust/docker-magento/assets/22292287/79286766-7292-4e25-9766-c6e9b1623e70">


Also, I've tried hitting "Forgot your password" and when I enter my email  shanemcgeown@gmail.com (the same one  I used running this commmand - bin/magento admin:user:create) I'm not receiving  anything  in my inbox ?  hmm ...

**Reply by YevhenZvieriev** [reply]
1. Due to security reasons, Magento has introduced account-disabled functionality while considering the security of the business.  
Try to execute this command `bin/magento admin:user:unlock Smgy94`.

2. > Also, I've tried hitting "Forgot your password" and when I enter my email [shanemcgeown@gmail.com](mailto:shanemcgeown@gmail.com) (the same one I used running this commmand - bin/magento admin:user:create) I'm not receiving anything in my inbox ? hmm ...

If you mean will you be able to get the letter via personal email, the answer is NO!

But if you pay attention to `compose.yaml` there is `mailcatcher` with 1080 port.

So, it intercepts all email letters and you can access this by following:

`http://magento.test:1080`

**Reply by User9511** [reply]
Thanks yes I was using that command to unlock the account. 

By going to this http://magento.test:1080 and manually resetting my password you have solved my issue!

Thanks so  much for your help!
